test/utilities/apexUtils.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  TextEditor
+} from 'wdio-vscode-service';
+import {
+  runCommandFromCommandPrompt
+} from './commandPrompt';
+import {
+  pause
+} from './miscellaneous';
+
+export async function createApexClassWithTest(name: string): Promise<void> {
+  const workbench = await browser.getWorkbench();
+  let textEditor: TextEditor;
+
+  // Using the Command palette, run SFDX: Create Apex Class to create the main class
+  const inputBox = await runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+  // Set the name of the new Apex Class
+  await inputBox.setText(name);
+  await inputBox.confirm();
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+  await pause(1);
+
+  // Modify class content
+  const editorView = workbench.getEditorView();
+  textEditor = await editorView.openEditor(name + '.cls') as TextEditor;
+  const classText = [
+    `public with sharing class ${name} {`,
+    `\tpublic static void SayHello(string name){`,
+    `\t\tSystem.debug(\'Hello, \' + name + \'!\');`,
```

Comment:
Line 39: it appears the single quotes are escaped, but with a back tick, they don't need to be escaped.  The line should be:
```
`\t\tSystem.debug('Hello, ' + name + '!');`,
```